### ISSUE TITLE
`ter-newline-after-var`: Doesn't detect new-line

### ISSUE THREAD

**Issue by wolfy1339** [buzinas/tslint-eslint-rules#366]
Take the following code, run it through tslint using the `"ter-newline-after-var": [true, "always"]` configuration. TSLint will then complain that there is no new-line after the variable declaration even though there is
```ts
let pendingStatus: string[] = []; // contains all pending checks from travis as multiple are sent

/**
 * @param  {string} s The commit hash string
 * @return {string}   Returns formatted commit hash string
 */
function fmt_url(s: string): string {
  return `\x0302\x1F${s}\x0F`;
}
```
